src/xml/zcl_abapgit_xml.clas.testclasses.abap

```diff
@@ -0,0 +1,90 @@
+CLASS ltcl_xml DEFINITION DEFERRED.
+
+CLASS ltcl_xml_concrete DEFINITION FOR TESTING
```

Comment:
suggest making ltcl_xml and cl_abapgit_xml FRIENDS instead, see https://answers.sap.com/questions/7607378/index.html

Reply:
do you suggest I remove ltcl_xml_concrete? I can't because cl_abapgit_xml is abstract, it can't be instantiated, even by using friendship.

Reply:
oh, okay 👍